README.md

```diff
@@ -1,13 +1,50 @@
-# bookstore
+### Bookstore application
+This application is supposed to be used for selling books. 
+Admin users have rights to uploads information about books and normal users can buy them.
+Normal users can put chosen books into carts. 
+Admin users also have rights to upload book categories, update and delete data about books and categories. 
+Books can belong to the categories depending on the book subject.
+
+## Technologies
+The application was made with technologies:
+1. Spring Boot
+2. Spring Security
+3. Spring Data JPA
+4. Hibernate
+5. Liquibase 
+6. Json Web token 
+7. Lombok 
+8. Mapstruct
+9. Test containers
+10. Springdoc
+
+## Controllers
+There are controllers:
+1. AuthenticationController. 
+This controller provides the way of registration of user and signing in of registered user.
+2. BookController. This controller handles CRUD operations with books.
+3. CartController. This controller is responsible for CRUD operations with cart.
+4. CategoryController. This controller handles CRUD operations with book categories.
+5. OrderController. Responsible for CRUD operations with book orders.
+
+## Launch the project locally
 
 Running the server for the first time:
-1. Clone the project
+1. Clone the project from the GitHub repository: https://github.com/artemklishch/bookstore
 2. Remove './target' directory if it exists with the command 'rm -rf target' (for the first running the server)
```

Comment:
this step is redundant as mvn clean automatically removes it

Reply:
++, updated